### PR TITLE
Edit team name

### PR DESCRIPTION
**Pull request by aabidshamji**
Added ability for the team leader to edit the team name

### CODE REVIEW

File: Pods/GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLPrioritizer.m [hunk 1]
```diff
@@ -0,0 +1,188 @@
+/*
```
Comment (samisuteria) [inline Pods/GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLPrioritizer.m:1]: why were these files updated?

File: Steps4Impact.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -794,7 +794,7 @@
 					};
 					269E34C51E765828000726C2 = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = 6SQ8F32998;
+						DevelopmentTeam = 479QYREFPY;
```
Comment (samisuteria) [inline Steps4Impact.xcodeproj/project.pbxproj:800]: please don't commit changes to project.pbxproj that effect build settings like dev team or bundle ids

File: Steps4Impact/App/AppController.swift [hunk 3]
```diff
@@ -213,6 +214,8 @@ class AppController {
     AKFCausesService.performAPIHealthCheck { (result) in
       switch result {
       case .failed:
+        let loginManager = LoginManager()
```
Comment (samisuteria) [inline Steps4Impact/App/AppController.swift:217]: this is technically a part of another bug but that's fine - just leave it in the comments of a PR

File: Steps4Impact/Supporting Files/Steps4Impact.entitlements [hunk 4]
```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>aps-environment</key>
```
Comment (samisuteria) [inline Steps4Impact/Supporting Files/Steps4Impact.entitlements:5]: this shouldn't be deleted

**Review by samisuteria — CHANGES_REQUESTED** [review]

File: Pods/GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLPrioritizer.m (hunk 1 above)
Comment (aabidshamji) [reply Pods/GoogleDataTransportCCTSupport/GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTFLLPrioritizer.m:1]: Shouldn't have been.

**Comment by aabidshamji** [comment]
Made necessary changes

**Comment by samisuteria** [comment]
closing in favor of: https://github.com/AgaKhanFoundation/WCF-iOS/pull/341

**Pull request closed without merging**